yt/geometry/particle_geometry_handler.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, order1, order2, filename, mutable_index):
         self._order2_orig = order2
         self.filename = filename
         self.mutable_index = mutable_index
+        self.loaded = False
```

Comment:
This doesn't seem like something we'd need to expose
```suggestion
        self._is_loaded = False
```